### DISCUSSION TITLE
제페토 커스텀 쉐이더 관련 문의

### DISCUSSION THREAD

**Discussion by ghost** [naverz/zepeto-studio-kor#801] [5. Q&A]
안녕하세요.

제페토 커스텀 쉐이더를 받아서 적용하려고 하는데
공식 가이드를 보고 여러번 따라 해봐도 계속 흰색 머터리얼이 되어버립니다.

유니티 기본 쉐이더는 적용이 잘 되는데
Standard_js, StandardTransparent_js 쉐이더만 저렇게 되는데 왜 그런건가요??

답변 부탁드립니다~

![image](https://user-images.githubusercontent.com/106231295/170209440-a9efc697-b9a7-4e0a-b75e-e67efaa34c7b.png)

![image](https://user-images.githubusercontent.com/106231295/170209026-a1c069c2-8b8e-44cb-bfe5-e8e60c9b13ad.png)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요~
월드 개발 이외의 문의 사항은 제페토 고객센터를 이용해 주시기를 부탁 드립니다. 
https://support.zepeto.me/hc/ko
감사합니다.
